{{cookiecutter.project_slug}}/core/management/commands/flutter.py

```diff
@@ -1834,82 +624,77 @@ def call_methods(self, options):
         """
 
         if options["main"]:
-            self.__replace_main()
+            self._replace_main()
             return
         elif options["yaml"]:
-            self.__add_packages()
+            self._add_packages()
             return
         elif options["routers"]:
-            self.__create_named_route()
+            self._build_named_routes()
             return
         elif options["clear"]:
-            self.__clear_project()
+            self._clear_project()
             sys.exit()
 
         else:
-            self.__init_flutter()
-            self.__build_settings_controller()
-            self.__build_utils()
-            self.__build_user_interface()
-            self.__build_custom_colors_file()
-            self.__build_custom_style_file()
-            self.__build_sized_extensions()
-            self.__build_string_extensions()
-            self.__build_custom_dio()
-            self.__build_custom_dio_interceptors()
-            self.__build_internationalization()
-            self.__build_logger_file()
-            self.__build_auth_app()
-            self.__build_flutter()
-            self.__create_named_route()
-            self.__create_exception_class()
+            self._init_flutter()
```

Comment:
Será que ficaria melhor, antes do início do projeto poderia checar se o Flutter está instalado, para evitar erros?